docs/DeveloperGuide.md

```diff
@@ -1,5 +1,6 @@
 ---
-layout: page title: Developer Guide
+layout: page 
+title: Developer Guide
 ---
 
 * Table of Contents {:toc}
```

Comment:
The table of contents is not correct. It will show as a bullet point in this format.
```suggestion
* Table of Contents
{:toc}
```